### PR TITLE
Harden tmbitwarden

### PR DESCRIPTION
**Pull request by szaimen**

### CODE REVIEW

**Comment by szaimen** [comment]
the check has worked in my testing. hence marking as ready for preview.

File: apps/tmbitwarden.sh [hunk 1]
```diff
@@ -18,6 +17,25 @@ debug_mode
 # Check if root
 root_check
 
+# Check if Bitwarden is already installed
+print_text_in_color "$ICyan" "Checking if Bitwarden is already installed..."
+if [ "$(docker ps -a >/dev/null 2>&1 && echo yes || echo no)" == "yes" ]
+then
+    if docker ps -a --format '{{.Names}}' | grep -Eq "bitwarden";
```
Comment (enoch85) [inline apps/tmbitwarden.sh]: does `is_this_docker_installed` work here?

**Comment by enoch85** [comment]
Much easier to follow your changes now.

I think this looks good, just want to know if we can use the existing function or not.

File: apps/tmbitwarden.sh (hunk 1 above)
Comment (enoch85) [reply apps/tmbitwarden.sh]: Nope, since it needs to be specific, like 'bitwarden/admin' for example

**Pull request merged**